yt_dlp/extractor/steam.py

```diff
@@ -53,96 +53,62 @@ class SteamIE(InfoExtractor):
             'playlistend': 2,
         }
     }, {
-        'url': 'http://steamcommunity.com/sharedfiles/filedetails/?id=242472205',
+        'url': 'https://store.steampowered.com/app/271590/Grand_Theft_Auto_V/',
         'info_dict': {
-            'id': 'X8kpJBlzD2E',
+            'id': '256757115',
+            'title': 'Grand Theft Auto V video 256757115',
             'ext': 'mp4',
-            'upload_date': '20140617',
-            'title': 'FRONTIERS - Trapping',
-            'description': 'md5:bf6f7f773def614054089e5769c12a6e',
-            'uploader': 'AAD Productions',
-            'uploader_id': 'AtomicAgeDogGames',
-        }
-    }]
+            'thumbnail': r're:^https://cdn\.[^\.]+\.steamstatic\.com',
+            'n_entries': 20,
+        },
+    }
+    ]
 
     def _real_extract(self, url):
         m = self._match_valid_url(url)
-        fileID = m.group('fileID')
-        if fileID:
-            videourl = url
-            playlist_id = fileID
-        else:
-            gameID = m.group('gameID')
-            playlist_id = gameID
-            videourl = self._VIDEO_PAGE_TEMPLATE % playlist_id
+        gameID = m.group('gameID')
+        playlist_id = gameID
```

Comment:
```suggestion
        playlist_id = self._match_valid_url(url).group('gameID')
```

what ae the other groups in the VALID_URL for?